plugins/amqp/src/plugin.cpp

```diff
@@ -279,6 +280,28 @@ class amqp_engine {
     return to_error(status);
   }
 
+  /// Consumes frames from broker, simply for the side effect of processing
+  /// heartbeats implicitly. Required if otherwise no interaction with the
+  /// broker would occur.
+  auto handle_heartbeat(operator_control_plane& ctrl) {
+    amqp_frame_t frame;
+    // We impose no timeout, either there is something to read or not. Never
+    // block!
+    struct timeval tv = {0, 0};
+    int status;
+    if (conn_ == nullptr)
+      return;
```

Comment:
```suggestion
    if (conn_ == nullptr) {
      return;
    }
```

Maybe you could run `clang-format` over you branch before submitting a PR? That would make sure it's already in the final expected format.

Reply:
Will do! Thanks

Reply:
This will change a lot of the existing code, actually.